pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -90,14 +91,13 @@ func memberTokenRunE(cmd *cobra.Command, args []string) error {
 		if file, err = os.OpenFile(memberTokenOpts.output, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600); err != nil {
 			return err
 		}
+		defer file.Close()
 	}
-	defer file.Close()
```

Comment:
Will it cause any issue without your fix?

Reply:
I didn't actually execute it to get an error. But according to the codes, the file will be nil when the output format parameter is empty. and it will return invalid argument if `file.Close()` is called.